css/style.css

```diff
@@ -558,3 +558,284 @@ textarea.form-element {
   border-top-left-radius: 0;
   border-top-right-radius: 0;
 }
+
+/* Add Desktop version Styling */
+
+@media (min-width: 768px) {
+  .menu {
+    display: none;
+  }
+
+  body {
+    position: relative;
+    width: 1440px;
+    height: 916px;
+    left: 0;
+    top: 1px;
+  }
+
+  .portfolio,
+  .about,
+  .contact {
+    display: flex;
+  }
+
+  .menu-font {
+    text-decoration: none;
+    font-family: Poppins, Georgia, 'Times New Roman', Times, serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    align-items: center;
+    text-align: center;
+    color: #344563;
+  }
+
+  .logo {
+    flex-grow: 0.9;
+    margin-left: 150px;
+  }
+
+  .contact {
+    margin-right: 140px;
+  }
+
+  .primary-text {
+    width: 500px;
+    height: 122px;
+    font-size: 48px;
+    line-height: 60px;
+    align-items: center;
+    letter-spacing: 0.37px;
+    margin-bottom: 6px;
+  }
+
+  .support-text-headline {
+    width: 920px;
+    height: 98px;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+  }
+
+  .desktop {
+    display: grid;
+    width: 80%;
+    padding: 0;
+    margin-left: -50px;
+  }
+
+  .desktop2 {
+    display: grid;
+  }
+
+  .tonicA,
+  .tonicB,
+  .canopyM,
+  .backdevM,
+  .tonicM,
+  .paraM {
+    display: none;
+  }
+
+  .can-desktop {
+    display: flex;
+    width: 83px;
+    margin-right: 0;
+  }
+
+  .backdev-desktop {
+    display: flex;
+    width: 105px;
+    margin-right: 0;
+  }
+
+  .tonic-desktop {
+    display: flex;
+    line-height: 52px;
+    color: #091e42;
+    font-weight: 700;
+  }
+
+  .uber-width {
+    width: 44px;
+  }
+
+  .lead-width {
+    width: 115px;
+  }
+
+  .para-desktop {
+    display: flex;
+    color: #344563;
+    width: 93%;
+    align-items: flex-start;
+  }
+
+  .htm-desktop {
+    display: flex;
+    width: 86px;
+  }
+
+  .grid-container {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    width: 80%;
+    padding: 0%;
+    margin-left: 10%;
+    grid-template-areas:
+      "item1 item2"
+      "item4 item3"
+      "item5 item6"
+      "item8 item7";
+  }
+
+  .grid-item1 {
+    border-top-right-radius: 0;
+    border-bottom-right-radius: 0;
+    border-bottom-left-radius: 1rem;
+    grid-area: item1;
+    margin-bottom: 5rem;
+  }
+
+  .grid-item2 {
+    border-top-left-radius: 0;
+    border-bottom-left-radius: 0;
+    border-top-right-radius: 1rem;
+    grid-area: item2;
+    margin-bottom: 5rem;
+  }
```

Comment:
- kindly fix these issues in all of your projects cards to match the Figma design : 

- [ ] kindly remove the space that divides the card image and description
- [ ] the technologies list should have a background of `#EBEBFF` instead of `rgb(231, 228, 228)` kindly fix it
- [ ] kindly reduce the space between the technologies and the project description
- [ ] kindly add some space between the button and the bottom of the cards


**_Figma design :_**
![image](https://user-images.githubusercontent.com/89282221/200124616-9543ea43-9e3a-4ae5-aaef-1cade8691a46.png)


**_your design:_**

![image](https://user-images.githubusercontent.com/89282221/200124624-030d102d-ad42-4ec6-9725-3a8facb770ba.png)